src/components/renders/bootstrap/select/select-render.tsx

```diff
@@ -79,16 +79,29 @@ export class SelectRender implements Renderer {
         }
       };
 
-      return [
-        <gx-bootstrap />,
-        <select {...attris}>
-          {this.options.map(({ disabled, innerText, selected, value }) => (
-            <option disabled={disabled} selected={selected} value={value}>
-              {innerText}
-            </option>
-          ))}
-        </select>
-      ];
+      return this.component.suggest
+        ? [
+            <gx-bootstrap />,
+            <input list={`${this.selectId}__datalist`}></input>,
+
+            <datalist id={`${this.selectId}__datalist`}>
```

Comment:
Imagine that in the future we want to change the suffix, we would have to change it in two places. What can we do to avoid having that duplication?

Reply:
You are right, it's better to save that string in a variable and just edit it in one place. DRY principle.
Check 0df5ffc commit!